backend/settings.js

```diff
@@ -90,6 +88,18 @@ function getDerivedSetting(settingName) {
                 'settings.yaml'
             );
 
+        case 'KEY_FILE':
+            return path.join(
+                getSetting('STORAGE_PATH'),
+                'privkey.pem'
```

Comment:
I've seen both in the humbly small time I've worked with certs. Letsencrypt generates a `privkey.pem` on the bot so I kind of just kept the same name throughout.